modules/kubernetes/ingress-nginx/templates/values.yaml.tpl

```diff
@@ -44,6 +44,8 @@ controller:
     http-snippet: |
       ${http_snippet}
     %{~ endif ~}
+    allow-snippet-annotations: ${allow_snippet_annotations_enabled}
+    annotation-value-word-blocklist: load_module,lua_package,_by_lua,location,root,proxy_pass,serviceaccount,{,},',\
```

Comment:
This seems dangerous. Is it tested?

Reply:
Yes, look at comments earlier in the PR and you see how the config respons to the same config defined in the hackerone example.

Reply:
On the other hand I'm a bit confused on how it works. The disable flag only seems to be on custom server annotation and not things like the whitelist annotation. I can read some more code and be sure that i have understood when it applies and not. I do think this is what we want but it's never bad to understand more :)